stripe/util.py

```diff
@@ -130,105 +130,22 @@ def secure_compare(val1, val2):
-        api_resources.Person.OBJECT_NAME: api_resources.Person,
-        api_resources.Plan.OBJECT_NAME: api_resources.Plan,
-        api_resources.Product.OBJECT_NAME: api_resources.Product,
-        api_resources.radar.EarlyFraudWarning.OBJECT_NAME: api_resources.radar.EarlyFraudWarning,
-        api_resources.radar.ValueList.OBJECT_NAME: api_resources.radar.ValueList,
-        api_resources.radar.ValueListItem.OBJECT_NAME: api_resources.radar.ValueListItem,
-        api_resources.Recipient.OBJECT_NAME: api_resources.Recipient,
-        api_resources.RecipientTransfer.OBJECT_NAME: api_resources.RecipientTransfer,
-        api_resources.Refund.OBJECT_NAME: api_resources.Refund,
-        api_resources.reporting.ReportRun.OBJECT_NAME: api_resources.reporting.ReportRun,
-        api_resources.reporting.ReportType.OBJECT_NAME: api_resources.reporting.ReportType,
-        api_resources.Reversal.OBJECT_NAME: api_resources.Reversal,
-        api_resources.Review.OBJECT_NAME: api_resources.Review,
-        api_resources.sigma.ScheduledQueryRun.OBJECT_NAME: api_resources.sigma.ScheduledQueryRun,
-        api_resources.SKU.OBJECT_NAME: api_resources.SKU,
-        api_resources.Source.OBJECT_NAME: api_resources.Source,
-        api_resources.SourceTransaction.OBJECT_NAME: api_resources.SourceTransaction,
-        api_resources.Subscription.OBJECT_NAME: api_resources.Subscription,
-        api_resources.SubscriptionItem.OBJECT_NAME: api_resources.SubscriptionItem,
-        api_resources.SubscriptionSchedule.OBJECT_NAME: api_resources.SubscriptionSchedule,
-        api_resources.SubscriptionScheduleRevision.OBJECT_NAME: api_resources.SubscriptionScheduleRevision,
-        api_resources.TaxId.OBJECT_NAME: api_resources.TaxId,
-        api_resources.TaxRate.OBJECT_NAME: api_resources.TaxRate,
-        api_resources.ThreeDSecure.OBJECT_NAME: api_resources.ThreeDSecure,
-        api_resources.Token.OBJECT_NAME: api_resources.Token,
-        api_resources.Topup.OBJECT_NAME: api_resources.Topup,
-        api_resources.Transfer.OBJECT_NAME: api_resources.Transfer,
-        api_resources.UsageRecord.OBJECT_NAME: api_resources.UsageRecord,
-        api_resources.UsageRecordSummary.OBJECT_NAME: api_resources.UsageRecordSummary,
-        api_resources.WebhookEndpoint.OBJECT_NAME: api_resources.WebhookEndpoint,
-        api_resources.terminal.Location.OBJECT_NAME: api_resources.terminal.Location,
-        api_resources.terminal.ConnectionToken.OBJECT_NAME: api_resources.terminal.ConnectionToken,
-        api_resources.terminal.Reader.OBJECT_NAME: api_resources.terminal.Reader,
-    }
+    from stripe.object_classes import OBJECT_CLASSES
+    return OBJECT_CLASSES
 
 
 def convert_to_stripe_object(
     resp, api_key=None, stripe_version=None, stripe_account=None
 ):
-    global OBJECT_CLASSES
-
-    if not OBJECT_CLASSES:
-        load_object_classes()
-    types = OBJECT_CLASSES.copy()
+    # Can we remove this copy?
+    types = get_object_classes().copy()
```

Comment:
Yeah, I think you can simply remove the `load_object_classes()` and `get_object_classes()` functions and just use `stripe.object_classes.OBJECT_CLASSES` in place of `types` in the `klass = types.get(...)` call below.

I wouldn't consider the removal of `load_object_classes()` a breaking change. It's technically public (but then again what isn't public in a Python package 🤔) but it's clearly an implementation detail that was never meant to be used by external users. 

We could simply have it return `stripe.object_classes.OBJECT_CLASSES` if you'd rather keep it for now though.

Reply:
cool I'm happy to remove `load_object_classes()`!

I'd rather keep `def get_object_classes()` though just because it's so strange to import within a function like that. Is that okay?

I think I can refactor away the `types` variable and the copy...

Reply:
(I don't think we can `return stripe.object_classes.OBJECT_CLASSES` without circular dep issues... when I tried, I got `AttributeError: module 'stripe' has no attribute 'object_classes'` ... I don't think we really want to deliberately expose `object_classes` on the root stripe object anyway)